.github/workflows/merge_to_master.yml

```diff
@@ -4,7 +4,7 @@ name: update_robottelo_image
 on:
   push:
     branches:
-      - master
+      - 6.9.z
```

Comment:
Maybe this wasn't meant to be? You might want to try resetting your branch with master

Reply:
oh dear

Reply:
Was caused by a failed rebase to master. I deleted and recreated the branch. Thank you.